hw2skeleton/pdb_to_fasta.py

```diff
@@ -0,0 +1,37 @@
+import sys
```

Comment:
Not everyone wants to go the FASTA route so I don't think we should add this file to this PR.